src/Loggable/Document/LogEntry.php

```diff
@@ -24,6 +24,8 @@
  *         @MongoODM\Index(keys={"username"="asc"})
  *     }
  * )
+ *
+ * @final since gedmo/doctrine-extensions 3.x
```

Comment:
I think this one shouldn't be `final` since is a Document and it could be proxied